src/redux/lhStory/lhStory.js

```diff
@@ -32,8 +51,28 @@ const listStory = () => (dispatch) => {
   dispatch({type: LH_STORY_LIST, list})
 }
 
+const getLHTransactions = (account, toBlock) => (dispatch) => {
+  dispatch({ type: LH_STORY_TRANSACTIONS_FETCH })
```

Comment:
LH_STORY_LIST was created specially for logic, that are now written in LH_STORY_TRANSACTIONS...